src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -391,6 +407,9 @@ message JobDescriptor {
 
     /// (Optional) Job disruption budget. If not defined, a job type specific (batch or service) default is set.
     JobDisruptionBudget disruptionBudget = 10;
+
+    /// (Optional) Networking configuration. If not defined, sane defaults are provided by the backend.
```

Comment:
Does proto3 have the idea of "not provided"?

Reply:
yea no I guess not :(

Reply:
Er yes, the 0 UNKNOWN is apparently best practice

Reply:
Message fields in Proto3 are nullable (presence), but scalar fields are not.